fortune.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/local/bin/python3.7
```

Comment:
That should really be `/usr/bin/env python`.

I don't see anything Python 3.7 specific, but I wouldn't merge anything that wouldn't support Python 2.7 and 3.5 and upwards.

Reply:
Makes sense, but shouldn't it be 

`/usr/bin/env python3`

Reply:
If the script is python3 only, then yes. Otherwise, just `python` could refer to either one.